sphinx/source/bokeh_theme/layout.html

```diff
@@ -28,32 +28,55 @@
   –––––––––––––––––––––––––––––––––––––––––––––––––– -->
   <link rel="stylesheet" href="{{ pathto('_static/' + theme_css, 1) }}">
 
+{%- macro css() %}
+    <link rel="stylesheet" href="{{ pathto('_static/' + style, 1) }}" type="text/css" />
+    <link rel="stylesheet" href="{{ pathto('_static/pygments.css', 1) }}" type="text/css" />
+    {%- for css in css_files %}
+      {%- if css|attr("filename") %}
+    {{ css_tag(css) }}
+      {%- else %}
+    <link rel="stylesheet" href="{{ pathto(css, 1) }}" type="text/css" />
+      {%- endif %}
+    {%- endfor %}
+{%- endmacro %}
+
+{%- block css %}
+{{- css() }}
+{%- endblock %}
+
   <!-- Scripts
   –––––––––––––––––––––––––––––––––––––––––––––––––– -->
 
-  <script src="//ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js"></script>
+{%- macro script() %}
+  <script type="text/javascript" id="documentation_options" data-url_root="{{ pathto('', 1) }}" src="{{ pathto('_static/documentation_options.js', 1) }}"></script>
+  {%- for js in script_files %}
+  {{ js_tag(js) }}
+  {%- endfor %}
+{%- endmacro %}
+
+{%- block scripts %}
   <script type="text/javascript">
-    var _BOKEH_CURRENT_VERSION = "{{ VERSION }}";
+   var _BOKEH_CURRENT_VERSION = "{{ VERSION }}";
   </script>
```

Comment:
This could just be removed I think? It only exists for our old version chooser which is being removed

Reply:
Gotcha! I wasn't sure if it was required or not.